src/components/ui/TextInput/index.js

```diff
@@ -28,17 +29,23 @@ export const TextInput = ({
     optionalOutput = <span class="optional">(Optional)</span>;
   }
 
+  let Component = 'input';
+  if (multiline) {
```

Comment:
👍 Ohhh, I like this.

Reply:
Hah, I definitely figured you'd hate that bit. 😜